.buildkite/pipeline.gpu_large.yml

```
@@ -84,6 +84,6 @@
     - PYTHON=3.8 DOC_TESTING=1 TRAIN_TESTING=1 TUNE_TESTING=1 ./ci/env/install-dependencies.sh
     - pip install -Ur ./python/requirements/ml/dl-gpu-requirements.txt
     - pip uninstall -y pytorch-lightning
-    - pip install lightning==2.0.0
+    - pip install lightning==2.0.4 pytorch-lightning==2.0.4  # todo move to requirements-test.txt
```

```suggestion
    - pip install lightning==2.0.4 pytorch_lightning==2.0.4  # todo move to requirements-test.txt
```

Thank you @krfricke. I checked when we install `lightning==2.0.0`, pip will always install the latest version of `pytorch_lightning`. That's why the library version has changed.